iceberg/bodo_iceberg_connector/schema.py

```diff
@@ -113,12 +112,7 @@ def get_iceberg_info(conn_str: str, schema: str, table: str, error: bool = True)
             iceberg_schema = None
             partition_spec = []
             sort_order = []
-
-            if warehouse is None:
-                raise IcebergError(
-                    "`warehouse` parameter required in connection string"
-                )
-            table_loc = gen_table_loc(catalog_type, warehouse, schema, table)  # type: ignore
+            table_loc = ""
```

Comment:
We don't need this anymore as AFAIK since we now create a transaction to get the write location